### PR TITLE
[14.0] shopfloor_mobile: checkout packaging list hide type

### PR DESCRIPTION
**Pull request by JuMiSanAr**
image

ref: rau-64

### CODE REVIEW

**Review by sebalix — APPROVED** [review]

File: shopfloor_mobile/static/wms/src/scenario/checkout.js [hunk 1]
```diff
@@ -283,15 +282,6 @@ const Checkout = {
                 },
             };
         },
-        select_delivery_packaging_manual_select_options: function () {
-            return {
-                showActions: false,
-                list_item_options: {
-                    loud_title: true,
-                    fields: [{path: "packaging_type"}],
```
Comment (simahawk) [inline shopfloor_mobile/static/wms/src/scenario/checkout.js:291]: Can't you simply remove the fields from here instead of dropping the method?

**Review by simahawk — CHANGES_REQUESTED** [review]
pls fix the commit msg being too long (see my update on PR title)

File: shopfloor_mobile/static/wms/src/scenario/checkout.js (hunk 1 above)
Comment (JuMiSanAr) [reply shopfloor_mobile/static/wms/src/scenario/checkout.js:291]: I've now removed the list_item_options entirely, as loud_title gave it an ugly look.

**Review by simahawk — APPROVED** [review]

**Pull request merged**